cmd/wallet-web/src/assets/img/credential--arrow-right-icon.svg

```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><g stroke="#190c21" stroke-width="1.5" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round"><path d="M5 12h12M13 17l5-5-5-5"/></g></svg>
```

Comment:
Is it not stored in git LFS?

Reply:
For instance, svgs I added in my PR were not showing like this 
<img width="1667" alt="image" src="https://user-images.githubusercontent.com/33902374/129992604-d1db4e07-c48f-4551-89e1-cf9c27dde686.png">

Reply:
Also, don't forget to optimize these svgs via https://jakearchibald.github.io/svgomg/

Reply:
SVG are optimized, i might not have lfs installed locally, thats why its showing like this i just copied stuff from my lost local pr and its still have to tested

Reply:
Please, install it and update this PR

Reply:
its updatted already